app/helpers/plantings_helper.rb

```diff
@@ -40,4 +40,18 @@ def display_planting(planting)
       "#{planting.owner}."
     end
   end
+
+  def plantings_active_tickbox_path(owner, show_all)
+    show_inactive_tickbox_path('plantings', owner, show_all)
+  end
+
+  def plantings_title(owner, crop)
```

Comment:
Turns out CodeClimate is more hardline than me, and wants you to unify them already :-/